patches/colorjs.io+0.4.5.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/node_modules/colorjs.io/dist/color.cjs b/node_modules/colorjs.io/dist/color.cjs
+index 3835d9a..5b8e8e1 100644
+--- a/node_modules/colorjs.io/dist/color.cjs
++++ b/node_modules/colorjs.io/dist/color.cjs
+@@ -2010,14 +2010,14 @@ var P3 = new RGBColorSpace({
+ // Default space for CSS output. Code in Color.js makes this wider if there's a DOM available
+ defaults.display_space = sRGB;
+ 
+-if (typeof CSS !== "undefined" && CSS.supports) {
++if (typeof CSS !== "undefined" && CSS?.supports) {
+ 	// Find widest supported color space for CSS
+ 	for (let space of [lab, REC2020, P3]) {
+ 		let coords = space.getMinCoords();
+ 		let color = {space, coords, alpha: 1};
+ 		let str = serialize(color);
+ 
+-		if (CSS.supports("color", str)) {
++		if (CSS?.supports("color", str)) {
```

Comment:
This one shouldn't need to be patched since by here `CSS.supports` exists and is not `null` due to the if statement above.